HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -250,10 +265,16 @@ def self.apply_room_air_conditioner(model, runner, cooling_system,
     control_zone.setSequentialHeatingFractionSchedule(ptac, get_sequential_load_schedule(model, 0))
 
     # Store info for HVAC Sizing measure
-    ptac.additionalProperties.setFeature(Constants.SizingInfoHVACCoolingCFMs, airflow_rate.to_s)
+    ptac.additionalProperties.setFeature(Constants.SizingInfoHVACRatedCFMperTonCooling, airflow_rate.to_s)
```

Comment:
Is this line and the next line intended to occur at the same time?

Reply:
Good catch, I will resolve it.